docs/configure

```diff
@@ -672,20 +676,20 @@ virtual_domains:
                               { and {{<{$spam_score_int}{${extract{sa_tag}{$address_data}}} } \
                                      {eq {1}{${extract{on_spamassassin}{$address_data}}}} \
                                     } \
-                              } { eq {0}{${extract{on_spamassassin}{$address_data}}}} \
+                              } { eq {0}{${extract{dom_spamassassin}{$address_data}}}} \
```

Comment:
These three don't seem right. I admit I haven't tried to parse this condition, but I doubt removing `on_spamassassin` is the right thing to do here.